docs/userguides/contracts.md

```diff
@@ -95,54 +95,64 @@ More information on publishing contracts can be found in [this guide](./publishi
 
 You can also use the [at() method](../methoddocs/contracts.html#ape.contracts.base.ContractContainer.at) from the same top-level project manager when you know the address of an already-deployed contract:
 
-```python
-from ape import project
+```{eval-rst}
+
+.. doctest::
 
-contract = project.MyContract.at("0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45")
+    >>> from ape import project
+    >>> contract = project.MyContract.at("0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45")
 ```
 
 ## From Any Address
 
 If you already know the address of a contract, you can create instances of it using the `Contract` top-level factory:
 
-```python
-from ape import Contract
+```{eval-rst}
+
+.. doctest::
 
-contract = Contract("0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45")
+    >>> from ape import Contract
+    >>> contract = Contract("0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45")
 ```
 
 It will fetch the `contract-type` using the explorer plugin from the active network, such as [ape-etherscan](https://github.com/ApeWorX/ape-etherscan).
 
 If you have the [ENS plugin](https://github.com/ApeWorX/ape-ens) installed, you can use `.eth` domain names as the argument:
 
-```python
-from ape import Contract
+```{eval-rst}
+
+.. doctest::
 
-contract = Contract("v2.registry.ychad.eth")
+    >>> from ape import Contract
+
+    >>> contract = Contract("v2.registry.ychad.eth")
```

Comment:
this will fail if `ape-ens` and `ape-etherscan` plugins are no installed, which i dont think they are in CI

Reply:
I will remove this doctest then.